GVFS/GVFS.Platform.Windows/WindowsFileSystem.cs

```diff
@@ -1,14 +1,19 @@
 ﻿using GVFS.Common;
 using GVFS.Common.FileSystem;
+using GVFS.Common.Tracing;
 using Microsoft.Win32.SafeHandles;
 using System;
+using System.ComponentModel;
 using System.IO;
 using System.Runtime.InteropServices;
+using static GVFS.Common.Git.LibGit2Repo;

```

Comment:
What was the reason for using `static` here?  Was it messy to use `LibGit2Repo` in the code below?